command/hcl2_upgrade.go

```diff
@@ -336,8 +337,14 @@ func transposeTemplatingCalls(s []byte) []byte {
 			return "${local.timestamp}"
 		},
 		"isotime": func(a ...string) string {
-			timestamp = true
-			return "${local.timestamp}"
+			if len(a) == 0 {
+				// returns rfc3339 formatted string.
+				return "${timestamp()}"
+			}
```

Comment:
Would it be a good idea to use the timestamp var here instead ?
```suggestion
			if len(a) == 0 {
				// returns rfc3339 formatted string.
				timestamp = true
				return "${local.timestamp}"
			}
```

Reply:
I _think_ the format of the timestamp variable is different. I considered making a new variable, but doing so would technically change the config behavior compared to the json configs, and felt more opinionated than we really need to be.

Reply:
Oh, right yes ! I wrongly though they were the same format because they were called the same.